CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Breaking changes
 
 ### New features & improvements
+- Add advanced scheduling option to the test configuration
```

Comment:
Don't we need the date and release version in changelog?

Reply:
I will add it to another MR with updated dependencies